content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -356,6 +356,43 @@ For example, a newly-created pod that is supposed to be able to reach both Pod A
 on Node 1 and Pod B on Node 2 may find that it can reach Pod A immediately,
 but cannot reach Pod B until a few seconds later.
 
+## NetworkPolicy and `hostNetwork` pods
+
+NetworkPolicy behaviour for `hostNetwork` pods is undefined, when
+1. a `hostNetwork` pod is selected by `spec.podSelector`.
+   
+   For example, if there is a `hostNetwork` pod on the nodeA (pod IP equals nodeA IP in that case), 
+   that has label `role: client`, the following NetworkPolicy:
+   ```yaml
+     ...
+     spec:
+       podSelector:
+         matchLabels:
+           role: client
+     ...
+   ```
+   may apply ingress and egress rules to all connections with nodeA IP, or just ignore
+   that pod as if it didn't match the selector, or do something else.
+2. a `hostNetwork` pod is selected by `ingress` or `egress` rules.
```

Comment:
```
2. a `hostNetwork` pod is selected by a `podSelector` in an `ingress` or `egress` rule.
```
(since there's the `ipBlock` case too, but you don't talk about that in this bullet point)